src/abstractdataframe/abstractdataframe.jl

```diff
@@ -106,18 +106,18 @@ Set column names
 names!(df::AbstractDataFrame, vals)
 ```
 
-### Arguments
+**Arguments**
```

Comment:
The consensus at https://github.com/JuliaLang/julia/pull/15136 appears to be to use `# Arguments`, with the idea that the doc system will eventually use the best heading style.